scene/gui/control.cpp

```diff
@@ -184,6 +184,14 @@ Size2 Control::_edit_get_minimum_size() const {
 }
 #endif
 
+void Control::reparent(Node *p_parent, bool p_keep_global_transform) {
+	Transform2D temp = get_global_transform();
+	Node::reparent(p_parent);
+	if (p_keep_global_transform) {
+		set_global_position(temp.get_origin());
```

Comment:
Should we also set `rotation` and `scale`?

Reply:
So, the funny thing is that Control has no `set_global_transform()` method and no methods for global rotation and global scale. Hence only position can be set easily.